src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -322,7 +322,6 @@ bool PolicyHandler::PolicyEnabled() const {
 
 bool PolicyHandler::LoadPolicyLibrary() {
   SDL_LOG_AUTO_TRACE();
-  ExchangePolicyManager(nullptr);
```

Comment:
@IGapchuk 
Why you need to remove it, I believe we can leave it in this

Reply:
@SNiukalov Was moved after `PolicyEnabled` check. Please take a look at 43aa7dd0a9f841a6ebfe08021d7e8dcff674821e